homeassistant/generated/integrations.json

```diff
@@ -811,6 +811,12 @@
       "config_flow": true,
       "iot_class": "cloud_polling"
     },
+    "bryant_evolution": {
+      "name": "Bryant Evolution",
+      "integration_type": "hub",
```

Comment:
is this really a hub, or a single device

Reply:
I *think* it's a hub, but I'm happy to discuss.

This is a single piece of hardware. It can connect to one or two different HVAC systems. Each system can have multiple zones. A (system, zone) maps to one ClimateEntity. 

It feels like a pice of hardware that exposes two different HVAC systems (say, an upstairs and downstairs furnace) would be a hub?

Reply:
the question is whether you setup the integration once per HVAC device, or once per system and the integration adds all climate devices at once. If at once -> hub, if one device -> device

Reply:
Ok, then I agree this is a device. Updated manifest.json.